sympy/physics/units/tests/test_util.py

```diff
@@ -22,42 +21,22 @@ def NS(e, n=15, **options):
 
 
 def test_dim_simplify_add():
-    with warns_deprecated_sympy():
-        assert dim_simplify(Add(L, L)) == L
-    with warns_deprecated_sympy():
-        assert dim_simplify(L + L) == L
+    # assert Add(L, L) == L
+    assert L + L == L
```

Comment:
Maybe I just don't understand the units code. Why do we have `length + length == length` and what does that mean?

Reply:
Oh does `length` just mean a generic quantity with dimensions of `length`?

Reply:
It's not a quantity,  it's a dimension. You can see dimensions as the space where quantities lie. 

If you add meters and feet the resulting object still represents a length. Some length plus length is still length.